### PR TITLE
Add "Community Examples" page, starting with Simply Square by NiVZ78

### PR DESCRIPTION
**Pull request by franc0is**
Original source in https://github.com/pebble/rockyjs/issues/35

### CODE REVIEW

File: examples/simple_square/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </p>
+    </div>
+
+    <div class="col-md-6">
+        <canvas id="pebble" class="rocky" width="432" height="504"></canvas>    
+    </div>
+    <div class="col-md-6">
+        <h2>What's going on?</h2>
+        <p>
+            WIP
```
Comment (franc0is) [inline examples/simple_square/index.html]: I'm not sure what we want to do here. Should every example come with a "What's going on" section?

**Comment by HBehrens** [comment]
In order to keep things simple, we might want to simply link to / embed JSBins/CodePens/JSFiddles whenever we receive those.

It would also make it simpler for contributors to add those to the repository as it's easier to copy&paste a snippet from a previous link in a list directly on Github than cloning, creating new directories and creating a "real" PR. What do you think @franc0is ?

**Comment by franc0is** [comment]
That could work. It also means we don't have to reformat them to meet pebble coding guidelines, which will save time :).

**Comment by HBehrens** [comment]
:) How do we proceed, will you change this PR?

**Comment by franc0is** [comment]
Yep. Sometimes today.

**Comment by franc0is** [comment]
@HBehrens updated to show community examples as JSBin embeds

File: examples/community.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <link href="../html/css/style.css" rel="stylesheet">
+  <!-- /build -->
+  <link href="common/css/style.css" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="page-header">
+    <h1>Community Examples</h1>
+    <p class="lead">
+    Here are a couple examples of what people are building with Rocky.js. Open a pull request to add yours!
```
Comment (HBehrens) [inline examples/community.html]: Strictly speaking this is just *one* example for now.

Also, what do you think about linking "open a pull request" to https://github.com/pebble/rockyjs/compare ?

File: examples/community.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    </p>
+  </div>
+  <div class="community-example">
+    <h3>Simply Square by <a class="example-author" href="https://github.com/NiVZ78">NiVZ78</a></h3>
+    <p><a class="jsbin-embed" href="http://jsbin.com/wakuha/embed?js,output&height=350px">
+      Simply Square by NiVZ78 on jsbin.com
+    </a></p>
+    <script src="https://static.jsbin.com/js/embed.min.js?3.35.9"></script>
+  <div>
+</div>
```
Comment (HBehrens) [inline examples/community.html]: if you add the

    

from the other pages it might encourage even more to fork / create PRs.

File: examples/community.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+<html lang = "en">
+<head>
+  <!-- build:css ../css/bootstrap.min.css -->
+  <link href="../html/css/bootstrap.min.css" rel="stylesheet">
+  <!-- /build -->
+  <!-- build:css ../css/style.css -->
+  <link href="../html/css/style.css" rel="stylesheet">
+  <!-- /build -->
+  <link href="common/css/style.css" rel="stylesheet">
+</head>
```
Comment (HBehrens) [inline examples/community.html]: I think we miss this at other places, too, but can you add a `<title>` tag?

File: examples/community.html (hunk 2 above)
Comment (franc0is) [inline examples/community.html]: I could reword it to say "This page is for examples" or something like
that. Using the singular is too depressing

Ok to link pull request creation

**Comment by HBehrens** [comment]
This looks great – I like that you also link to this new page from the root readme :)

A few nits :shipit:

File: examples/community.html (hunk 3 above)
Comment (franc0is) [inline examples/community.html]: Ok will do

File: examples/community.html (hunk 4 above)
Comment (franc0is) [inline examples/community.html]: Sounds good

File: examples/community.html (hunk 2 above)
Comment (HBehrens) [inline examples/community.html]: Also, a phrase that's not explicitly using singular helps us to not forget to update it in the future ;)

**Comment by franc0is** [comment]
Thanks for the review !

File: examples/community.html (hunk 2 above)
Comment (franc0is) [inline examples/community.html]: :white_check_mark:

File: examples/community.html (hunk 3 above)
Comment (franc0is) [inline examples/community.html]: :white_check_mark:

File: examples/community.html (hunk 4 above)
Comment (franc0is) [inline examples/community.html]: :white_check_mark:
